examples/testmonitor/results/Results.cs

```diff
@@ -0,0 +1,165 @@
+        /// </summary>
+        /// <param name="name">The test step's name.</param>
+        /// <param name="stepType">The test step's type.</param>
+        /// <param name="current">The value of the measured electrical current.</param>
+        /// <param name="voltage">The value of the measured electrical voltage.</param>
+        /// <param name="power">The value of calculated electrical power.</param>
+        /// <param name="lowLimit">The value of the low test limit.</param>
+        /// <param name="highLimit">The value of the high test limit.</param>
+        /// <returns>The <see cref="StepData"/> used to create a test step.</returns>
+        private static StepData GenerateStepData(string name, string stepType, double current, double voltage, double power, double lowLimit, double highLimit)
+        {
+            Random random = new Random();
+            var status = new Status(StatusType.Running);
+            var inputs = new List<NamedValue>();
+
+            var outputs = new List<NamedValue>();
+
+            var parameters = new List<Dictionary<string, string>>();
+            if (stepType.Equals("NumericLimit"))
+            {
+                // Add parameters to match the TestStand data model
+                var parameter = new Dictionary<string, string>();
+                parameter.Add("name", "Power Test");
+                parameter.Add("status", "status");
+                parameter.Add("measurement", $"{power}");
+                parameter.Add("units", null);
+                parameter.Add("nominalValue", null);
+                parameter.Add("lowLimit", $"{lowLimit}");
+                parameter.Add("highLimit", $"{highLimit}");
+                parameter.Add("comparisonType", "GELT");
+                parameters.Add(parameter);
+
+                // Record electrical current and voltage as inputs
+                inputs = new List<NamedValue>()
+                {
+                    new NamedValue("current", current),
+                    new NamedValue("voltage", voltage)
+                };
+
+                // Record electrical power as an output
+                outputs = new List<NamedValue>()
+                {
+                    new NamedValue("power", power)
+                };
+
+                // Set the step's status based on if the electrical power is within the test limits
+                status = (power < lowLimit || power > highLimit) ? new Status(StatusType.Failed) : new Status(StatusType.Passed);
+            }
+
+            var stepData = new StepData()
+            {
+                Inputs = inputs,
+                Name = name,
+                Outputs = outputs,
+                StepType = stepType,
+                Status = status,
+                TotalTimeInSeconds = random.NextDouble() * 10,
+                Parameters = parameters,
+                DataModel = "TestStand",
+            };
+
+            return stepData;
+        }
+
+        static void Main(string[] args)
```

Comment:
`Main` contains a lot of the contextual information like a description of what we're simulating. I wonder if it would improve readability to move `Main` above `GenerateStepData`? that way, it's easier to read the example from top to bottom.